packages/site/cms-site/src/previewskeleton/PreviewSkeleton.module.scss

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Again, i think eslint/prettier should prevent the first, empty line and the missing new line at the end of the file